client/jetpack-cloud/sections/partner-portal/stored-credit-card/index.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { PaymentLogo } from '@automattic/composite-checkout';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function StoredCreditCard( props ): ReactElement {
+	const translate = useTranslate();
+	const creditCard = props.card;
+
+	const expiry = new Date( creditCard.expiry );
+	const month = expiry.getMonth();
+	const year = expiry.getFullYear().toString().substr( 2, 2 );
+
+	return (
+		<div className="stored-credit-card">
+			<div className="stored-credit-card__header">
+				<div className="stored-credit-card__payment-logo">
+					<PaymentLogo brand={ creditCard.card_type } isSummary={ true } />
+				</div>
+
+				<div className="stored-credit-card__primary">{ translate( 'Primary' ) }</div>
```

Comment:
It doesn't look like there is any check if this is actually the "Primary card" of the Stripe account. I would say we need to check the customer object if the card matches the default source before it's actually "primary".

Reply:
Answered this in the comment below